apps/s_client.c

```diff
@@ -2942,6 +2948,13 @@ int s_client_main(int argc, char **argv)
                 goto shut;
             }
         }
+
+        /*don't wait for client input in the non-interactive mode*/
```

Comment:
spaces around the text:
`/* don't wait for client input in the non-interactive mode */`

Reply:
done.